lib/jekyll-mentions.rb

```diff
@@ -2,28 +2,36 @@
 require 'html/pipeline'
 
 module Jekyll
-  class Mentions < Jekyll::Generator
+  class Mentions < Jekyll::Converter
+    MENTIONABLE_EXTS = %w{.html .htm .md .markdown .mkdn .mkd .textile}
```

Comment:
related talk about this via @gjtorikian in jemoji https://github.com/jekyll/jemoji/pull/18#discussion_r29767713

we'll probably go with the same decision